.github/workflows/quickstart_microprofile-fault-tolerance_ci.yml

```diff
@@ -0,0 +1,14 @@
+name: WildFly microprofile-fault-tolerance Quickstart CI
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths:
+      - 'microprofile-fault-tolerance/**'
+      - '.github/workflows/quickstart_ci.yml'
+jobs:
+  call-quickstart_ci:
+    uses: ./.github/workflows/quickstart_ci.yml
+    with:
+      QUICKSTART_PATH: microprofile-fault-tolerance
+      MICROPROFILE: true
```

Comment:
this was changed, now it should be `TEST_BOOTABLE_JAR: true`

Reply:
Fixed.